aimon/_utils/_utils.py

```diff
@@ -394,3 +394,65 @@ def lru_cache(*, maxsize: int | None = 128) -> Callable[[CallableT], CallableT]:
         maxsize=maxsize,
     )
     return cast(Any, wrapper)  # type: ignore[no-any-return]
+
+
+def extract_response_metadata(user_query, user_instructions, response):
+
+    import logging
+
+    def get_source_docs(chat_response):
```

Comment:
this seems to be LlamaIndex specific code. Lets move it to aimon_llamaindex package?

Reply:
Ok